projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -3140,5 +3140,265 @@ public string FatherEthnicityLiteral
+                    name = new HumanName();
+                    name.Use = HumanName.NameUse.Official;
+                    name.Family = value;
+                    Attendant.Name.Add(name);
+                }
+            }
+        }
+
+        /// <summary>Attendant name.</summary>
+        /// <value>the attendant's name</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.AttendantName = "Janet Seito";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Attendants's Name: {ExampleBirthRecord.AttendantName}");</para>
+        /// </example>
+        [Property("Attendant Name", Property.Types.String, "Birth Certification", "Name of attendant.", true, VR.IGURL.Practitioner, true, 6)]
+        [FHIRPath("Bundle.entry.resource.where($this is Practitioner)", "name")]
+        public string AttendantName
+        {
+            get
+            {
+                if (Attendant != null && Attendant.Name != null)
+                {
+                    return Attendant.Name.First().Text;
+                }
+                return null;
+            }
+            set
+            {
+                if (Attendant == null)
+                {
+                    CreatePractitioner();
+                }
+                HumanName name = Attendant.Name.FirstOrDefault();
+                if (name != null && !String.IsNullOrEmpty(value))
+                {
+                    name.Text = value;
+                }
+                else if (!String.IsNullOrEmpty(value))
+                {
+                    name = new HumanName();
+                    name.Use = HumanName.NameUse.Official;
+                    name.Text = value;
+                    Attendant.Name.Add(name);
+                }
+            }
+        }
+
+        /// <summary>Attendants NPI</summary>
+        /// <value>the attendants npi</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.AttendantNPI = "1234567890";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Attendants NPI: {ExampleBirthRecord.AttendantNPI}");</para>
+        /// </example>
+        [Property("Attendants NPI", Property.Types.String, "Birth Certification", "Attendant's NPI.", true, VR.IGURL.Practitioner, true, 13)]
+        [FHIRPath("Bundle.entry.resource.where($this is Practitioner).identifier.where(system='http://hl7.org/fhir/sid/us-npi')", "value")]
+        public string AttendantNPI
+        {
+            get
+            {
+                return Attendant?.Identifier?.Find(id => id.System == "http://hl7.org/fhir/sid/us-npi")?.Value;
+            }
+            set
+            {
+                if (Attendant == null)
+                {
+                    CreatePractitioner();
+                }
+                if (Attendant.Identifier.Count > 0)
+                {
+                    Attendant.Identifier.Clear();
```

Comment:
Should there only every be one identifier? Does this need to be cleared?

Reply:
I believe so, in our IG under Key Elements, it says the cardinality for identifier:NPI is 0..1